#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-autobuild.dpatch by Daniel Baumann <daniel.baumann@panthera-systems.net>
##
## DP: Disable modules check to make it buildable without X11.

@DPATCH@

diff -Naur tinyerp-client-3.3.0.orig/setup.py tinyerp-client-3.3.0/setup.py
--- tinyerp-client-3.3.0.orig/setup.py	2006-05-18 16:13:15.000000000 +0000
+++ tinyerp-client-3.3.0/setup.py	2006-06-03 20:21:54.000000000 +0000
@@ -81,7 +81,7 @@
 Programming Language :: Python
 """
 
-check_modules()
+#check_modules()
 
 # create startup script
 start_script = \
